spec/fcontext_spec.rb

```diff
@@ -17,7 +17,7 @@ def converge_dsl(*recipes,&block)
 describe 'selinux_policy fcontext' do
 
   let :chef_run do
-    ChefSpec::SoloRunner.new(step_into: ['selinux_policy_fcontext']).converge_dsl('selinux_policy') do
+    ChefSpec::SoloRunner.new(platform: 'centos', version: 7.0, step_into: ['selinux_policy_fcontext']).converge_dsl('selinux_policy') do
```

Comment:
Hrm. At some point ChefSpec tests were failing without providing platform information. This does not appear to be the case any longer and I will revert the changes accordingly.